src/ngMessages/messages.js

```diff
@@ -629,10 +629,8 @@ angular.module('ngMessages', [], function initAngularHelpers() {
    * @scope
    *
    * @description
-   * `ngMessageExp` is a directive with the purpose to show and hide a particular message.
-   * For `ngMessageExp` to operate, a parent `ngMessages` directive on a parent DOM element
-   * must be situated since it determines which messages are visible based on the state
-   * of the provided key/value map that `ngMessages` listens on.
+   * `ngMessageExp` is the same as {@link module:ngMessage `ngMessage`} but instead
```

Comment:
Shouldn't this be `directive:ngMessage` or something (instead of `module:ngMessage`)?